### PR TITLE
fix: cf pages

### PR DESCRIPTION
**Pull request by millsp**
(mirror of https://github.com/prisma/ecosystem-tests/pull/4574)

### CODE REVIEW

File: driver-adapters-wasm/pg-cfpages-basic/run.sh [hunk 1]
```diff
@@ -7,5 +7,9 @@ pnpm install
...
 pnpm prisma generate
 
-# pnpm wrangler pages deploy . --project-name pg-cfpages-basic --node-compat | tee deployment-logs.txt
-# sleep 15
\ No newline at end of file
+# First build the functions using the `--node-compat` flag into the `_worker.js` directory
+# (Note that the functions are in the non-standard `fns` directory so that they are not confused with the output `_worker.js` directory.)
+pnpm wrangler pages functions build fns --node-compat --outdir _worker.js
+# Now deploy the _worker.js alongside the index.html asset to Pages
+pnpm wrangler pages deploy . --project-name pg-cfpages-basic -- | tee deployment-logs.txt
```
Comment (janpio) [inline driver-adapters-wasm/pg-cfpages-basic/run.sh]: Probably clearer to build into `build/_worker.js` and then to copy `index.html` to `build/index.html` to make clear that we are "creating" a pages compatible build in `build/` and then upload that via `wrangler pages deploy build`?

File: driver-adapters-wasm/pg-cfpages-basic/run.sh [hunk 2]
```diff
@@ -7,5 +7,9 @@ pnpm install
 
 pnpm prisma generate
```
Comment (janpio) [inline driver-adapters-wasm/pg-cfpages-basic/run.sh:9]: TODO here: Delete the file or folder that we later create via `wrangler pages functions build` so we can run the same command twice here, once for default output and once for custom output.

File: driver-adapters-wasm/pg-cfpages-basic/run.sh (hunk 1 above)
Comment (janpio) [inline driver-adapters-wasm/pg-cfpages-basic/run.sh]:
```suggestion
# Make sure build folder does not exist
rm -rf build
# Build pages function to build/_worker.js
pnpm wrangler pages functions build fns --node-compat --outdir build/_worker.js
# Copy index.html to build folder
cp index.html build/index.html
# Now deploy the _worker.js alongside the index.html asset to Pages
pnpm wrangler pages deploy ./build --project-name pg-cfpages-basic -- | tee deployment-logs.txt
```

Maybe?

Comment (millsp) [reply driver-adapters-wasm/pg-cfpages-basic/run.sh]: Will wrangler know that the folder is `build`?

Comment (millsp) [reply driver-adapters-wasm/pg-cfpages-basic/run.sh]: Or should this last line be `pnpm wrangler pages deploy ./build --project-name pg-cfpages-basic -- | tee deployment-logs.txt`

Comment (janpio) [reply driver-adapters-wasm/pg-cfpages-basic/run.sh]: Good catch, I missed that 👍

**Comment by petebacondarwin** [comment]
Magic!

**Pull request merged**